packages/typography/src/H1/H1.tsx

```diff
@@ -25,7 +25,6 @@ const H1 = Polymorphic<H1Props>(
 
     return (
       <Component
-        as="h1"
```

Comment:
Just trying to understand this - is this prop not necessary any more because we're using the `usePolymorphic` hook, which means the `as` prop is essentially handled on the parent element?

Reply:
Exactly. The `Component` component is already a `<Polymorph as={...} />` under the hood